tico/config/v1.py

```diff
@@ -20,6 +20,8 @@
 @dataclass
 class CompileConfigV1(CompileConfigBase):
     legalize_causal_mask_value: bool = False
+    convert_lhs_const_mm_to_fc: bool = False
+    convert_rhs_const_mm_to_fc: bool = True
```

Comment:
On second thought, just `convert_const_mm_to_fc` could be simpler choice. Do you have any reasons that chose this design?